packages/now-node/test/fixtures/04-tsx-resolve/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "strict": true,
     "esModuleInterop": true,
     "sourceMap": true,
+    "jsx": "react",
```

Comment:
Why is this necessary?

Reply:
https://github.com/vercel/vercel/pull/5143#discussion_r483290387